com.unity.render-pipelines.universal/ShaderLibrary/Lighting.hlsl

```diff
@@ -467,18 +466,34 @@ half3 HackSampleSH(half3 normalWS)
 half3 GlossyEnvironmentReflection(half3 reflectVector, half perceptualRoughness, half occlusion)
 {
 #if !defined(_ENVIRONMENTREFLECTIONS_OFF)
+#if REFLECTION_PROBE
```

Comment:
We should not have reflection probe as part of global quality, it's required for any IBL / Physically Based shading, quality is more related to how we blend these. 

Disabling reflection probe per-material it's still fine. it's per-object choice.